CMakeLists.txt

```diff
@@ -240,6 +240,26 @@ IF(LIKWID)
     TARGET_LINK_LIBRARIES(exadg ${LIKWID})
 ENDIF()
 
+# preCICE
+# the environment variable precice_DIR is searched by default
+FIND_PACKAGE(precice 2.0
+        HINTS ${precice_DIR} ${PRECICE_DIR} $ENV{PRECICE_DIR}
+  )
+# Pick up preCICE if we are able to locate it
```

Comment:
I am not sure if we want to have this behavior. I would prefer that the user always has to specify whether precice should be used or not, and that `FIND_PACKAGE` from above is shifted into the if-statement `IF(${EXADG_WITH_PRECICE))`. It should be done as for FFTW in my opinion.